docs/docs/getting-started/04-examples/02-editor-example.md

```diff
@@ -15,88 +15,120 @@ on the page to expand either the code editor or graph view to full screen.
 
-## Auto-lifecycle events
+![timer settings](./assets/timer-settings.png)
 
-![timer example](./assets/auto-lifecycle-events.png)
+### Auto-lifecycle events
 
 Because the timer components are lifecycle components, they only count down the time when they are active. By default,
 the "Load" or "Transition" events start the lifecycle component in an unconfigured state. The "auto-configure" and
-"auto-activate" toggle switches on the component can be used to enable or disable
-[auto lifecycle events](../../concepts/05-building-blocks/03-components.md#auto-lifecycle-events).
+"auto-activate" toggle switches can be used to enable or disable
+[auto-lifecycle events](../../concepts/05-building-blocks/03-components.md#auto-lifecycle-events).
 
 When both toggle switches are enabled, the YAML code will show the corresponding events for the componnts:
 
 ```yaml
     events:
-      is_unconfigured:
-        lifecycle: configure
-      is_inactive:
-        lifecycle: activate
+      transitions:
+        on_load:
+          lifecycle:
+            component: timer
+            transition: configure
+        on_configure:
+          lifecycle:
+            component: timer
+            transition: activate
 ```
 
-Disabling one or both auto event switches will also remove the corresponding events from the YAML, while enabling the
-switch will regenerate the event in the YAML.
+Disabling one or both auto-lifecycle event switches will also remove the corresponding events from the YAML, while
+enabling the switch will regenerate the event in the YAML.
+
+## Setting component parameters
+
+Component parameters can be edited directly in the settings panel. Change the value of the Timeout parameter and look
+for the corresponding change in the code editor. Then press play on the application and observe that the timer now has a
+different duration before triggering the "Is timed out" predicate.
+
+<!-- TODO: explain behavior with "default" once it's working as intended -->
+
+## Adding and deleting elements
+
+Elements can be added to the graph using the sidebar menu; press the (+) button in the top right corner of the graph to
+open the sidebar and see a list of available application elements. At the top of the list are the "Hardware Interface",
+"Trigger Events Button", "Sequence" and "Condition" nodes. These are followed by a list of all available components from
+AICA Core and any additionally installed packages, grouped by package. Clicking on any element in the sidebar will
+automatically add it to the graph.
+
+<!-- TODO: link the examples once they exist -->
 
-<!-- TODO: Replace the following sections with adding interactive buttons 
+To delete an element from the graph, press the small menu icon in its top right corner, then click Remove. Elements can
+also be deleted by selecting them with a click and pressing the Delete keyboard shortcut.
```

Comment:
```suggestion
also be deleted by selecting them with a click and pressing the Delete key on your keyboard.
```

Reply:
This is actually a good point because it's not correct...Non-mac keyboards have a backspace and a delete key whereas mac keyboards only have a delete. For non-mac, only the backspace key works

Reply:
huh, yes true. We should then probably mention it in a platform-specific way

Reply:
Even better, we take care of the keybindings explicitly in the future. For now, I made platform specific tabs

Reply:
nice, thanks! Should we through Windows in too (e.g., Linux/Windows)? :smile: or not just yet

Reply:
Not just yet :smile: